### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by zrocweb** [alkarinv/BattleArena#138]
This is a long shot, but I thought I would ask.  

To avoid an abundance of log messages as well as console messages and especially server load, is it possible to add something that will disable all scheduled events if no one logs into the server for 'x' hours.  This will help reduce 'overnight' issues when my servers are mostly dead due to people not being all at these hours of the night. 

Or maybe in addition or as a substitute would be to implement a method where the schedule can be auto d-activated between 'x' and 'x' hours.

Edit:
I am asking this because I noticed the past few days I had BA running with one event in the scheduler, I would repeatedly get warning's about the server be overloaded.  Its happening even with players not online.  I confirmed it was definitely BA because I disabled all the BAS tasks and have not received a warning message since.  Performed several restarts with and without BAS on/off.  With it on over a short period of time, receive the warnings, with it off, no warnings.  My server has over 1 gig of memory avail after a normal startup, so I know it's not a memory issues on my server, at least I believe it's not.

**Comment by alkarinv** [comment]
So I can probably add something in about not starting events if no one is online.

CPU issues when no one is online and no events are happening?

**Comment by zrocweb** [comment]
Yes.  Perhaps if no one is online for 'x' mins or secs cancel all scheduled events.  And then when there are > 1 players online after 'x' seconds, re-schedule the events.  Obviously the 'x' seconds needs to be config option as well as the ability to turn this option on/off ..
